dags.yaml

```diff
@@ -1359,3 +1359,19 @@ bqetl_crash:
     start_date: "2023-12-10"
   tags:
     - impact/tier_2
+
+bqetl_use_counter_analysis:
+  schedule_interval: 0 8 * * *
+  default_args:
+    depends_on_past: false
+    email:
+      - kwindau@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
```

Comment:
This would usually be set to false since we do not want to get an email on each retry of the job, rather only if all retries failed.

Reply:
good point, fixed this - thank you! :)